spec/01-unit/010-router_spec.lua

```diff
@@ -1344,7 +1304,7 @@ describe("Router", function()
 
         local router = assert(Router.new(use_case))
 
-        local _ngx = mock_ngx("GET", "/users/123/profile/hello/world", {})
+        local _ngx = mock_ngx("GET", "/users/123/profile/hello/world", { ["host"] = "domain.org" })
```

Comment:
Yeah... I tend to not follow the 80 chars rule (enough) in tests file. But you are right! And I do think this warrants a little cleanup! It's a quick job :)